cpp/src/parquet/column_writer_test.cc

```diff
@@ -649,7 +649,7 @@ TYPED_TEST(TestPrimitiveWriter, DictionaryFallbackVersion1_0) {
 }
 
 TYPED_TEST(TestPrimitiveWriter, DictionaryFallbackVersion2_0) {
-  this->TestDictionaryFallbackEncoding(ParquetVersion::PARQUET_2_4);
+  this->TestDictionaryFallbackEncoding(ParquetVersion::PARQUET_2_6);
```

Comment:
Emmm should we traverse all 2_0, 2_4, 2_6 ? Or just 2_0 is ok?

Reply:
I am working on having the tests cover at least 2.4 and 2.6! I started something locally, and will push once I have it built.

Reply:
```C++
this->TestDictionaryFallbackEncoding(ParquetVersion::PARQUET_2_4);
this->TestDictionaryFallbackEncoding(ParquetVersion::PARQUET_2_6);
```